pkg/appmgmt/general/general.go

```diff
@@ -90,7 +90,7 @@ func (os *Manager) Stop() {
 	// noop
 }
 
-func (os *Manager) getTaskMetadata(pod *v1.Pod) (interfaces.TaskMetadata, bool) {
+func (os *Manager) GetTaskMetadata(pod *v1.Pod) (interfaces.TaskMetadata, bool) {
```

Comment:
I tried moving both methods to `utils`, but ended up with circular deps. So leaving them here.